conan/tools/qbs/qbsdeps.py

```diff
@@ -166,6 +169,15 @@ def __init__(self, conanfile):
         :param conanfile: The current recipe object. Always use ``self``.
         """
         self._conanfile = conanfile
+        self._use_cmake_file_name = False
+
+    @property
+    def use_cmake_file_name(self):
+        return self._use_cmake_file_name
+
+    @use_cmake_file_name.setter
+    def use_cmake_file_name(self, value):
+        self._use_cmake_file_name = value
```

Comment:
Same as above, probably no need of new setter/getter, and the ``QBSDeps.set_property()`` is the way to go